src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -109,6 +112,10 @@ def __init__(self, url, max_schemas_per_subject=1000, ca_location=None, cert_loc
         self.url = utils.urldefragauth(self.url)
 
         self._session = s
+        key_password = conf.pop('ssl.key.password', None)
+        _https_session = self.make_https_session(ca_path, s.cert, s.auth, key_password)
```

Comment:
no need for this local variable. Can directly assign to `self._https_session`

Reply:
Sure!